src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/LicenceActivationService.kt

```diff
@@ -44,15 +44,28 @@ class LicenceActivationService(
 
   private fun determineLicenceEligibility(matchedLicences: List<LicenceWithPrisoner>): Pair<List<LicenceWithPrisoner>, List<Licence>> {
     val approvedHdcBookingIds = findBookingsWithHdc(matchedLicences)
-    val (ineligibleLicences, eligibleLicences) = matchedLicences.partition { approvedHdcBookingIds.contains(it.bookingId) }
+
+    // Filter out HDC licences that have not been approved for HDC as we don't want to deactivate them for MVP1
+    val filteredLicences =
+      matchedLicences.filterNot {
+        it.licence.kind === LicenceKind.HDC && !approvedHdcBookingIds.contains(it.bookingId)
+      }
+
+    val (eligibleLicences, ineligibleLicences) = filteredLicences.partition {
+      if (approvedHdcBookingIds.contains(it.bookingId) && it.licence.kind === LicenceKind.HDC) {
+        true
+      } else {
+        !approvedHdcBookingIds.contains(it.bookingId)
+      }
```

Comment:
very rarely need to use `===` in kotlin.
worth using static imports for this

could possibly make this a bit simpler: 
```kt
val approvedForHdc = approvedHdcBookingIds.contains(it.bookingId)
(it.licence.kind == HDC && approvedForHdc) || !approvedForHdc 
```